firmware/thruna/BMS/Src/App/App_Soc.c

```diff
@@ -1,5 +1,50 @@
 #include <math.h>
 #include "App_Soc.h"
+#include <stdint.h>
+#include <float.h>
+#include "App_SharedConstants.h"
+#include "App_SharedProcessing.h"
+#include "lut/App_Lut_CellVoltageToSoc.h"
+
+#define STATE_OF_HEALTH (1.0f)
```

Comment:
Will update this based on Edwin's experimental testing before merging, was told is closer to 0.93

Reply:
Will you be measuring state of health? Or just going off of Edwin's findings

Reply:
In the future yes, but don't have accurate enough data yet. I did get an estimated value of 0.946 from a charging cycle last Thursday, comparing the computed SOC to the value found on the LUT based on settled min-cell-voltage, but given my starting point for the computed SOC was close to the 5% garbage region, I don't know how accurate that is. Definitely need to do something about not having data below 5%.

Reply:
when you update it, please leave a comment on where the number comes from

Reply:
Edwin is unsure of where his State of Health number is doccumented, but believes it to be around 93%. Based on math from charging above, I got an estimated value of 94.46%. For now I used a value of 0.94, or 94% splitting the difference. This can be tuned in the future with more testing.